### PR TITLE
LiveView Form components & comverter

### PR DESCRIPTION
**Pull request by phgrey**

### CODE REVIEW

File: lib/moon_web/examples/light/form_example/registration.ex [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          <.input field={form[:username]} />
+        </.field>
+        <.field label="Label for TextArea" hint="Hint for TextArea" field={form[:email]}>
+          <.text_area field={form[:email]} />
+        </.field>
+        <.field field={form[:agrees_to_terms_of_service]}>
+          <.label field={form[:agrees_to_terms_of_service]} size="sm">
+            <Form.switch field={form[:agrees_to_terms_of_service]} /> I agree terms of service
+          </.label>
+        </.field>
```
Comment (gfrancischelli) [inline lib/moon_web/examples/light/form_example/registration.ex:55]: There are better alternatives to this style of API, see #810

Comment (phgrey) [reply lib/moon_web/examples/light/form_example/registration.ex:55]: > There are better alternatives to this style of API, see #810

@gfrancischelli, sorry, see what?

Comment (phgrey) [reply lib/moon_web/examples/light/form_example/registration.ex:55]: oh, got it, #814

File: docs/principes.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+
+## Our main goals are:
+- create multi-technology design system
+- ... to make frontend development easier for our consumers
+- ... and make library development transparent, involve library consumers in feature discussion and/or implementing
+
+## Programming rules:
+We do preffer and respect the following principes:
+Headless - as soon as we're using few techologies, we need to use headless approach, so sub-components structure & interactions are mostly predefined when creatig components for each technology.
```
Comment (gfrancischelli) [inline docs/principes.md:12]: "we're using few techologies" what does this mean?

Comment (phgrey) [reply docs/principes.md:12]: sorry, some kind of draft documentation is here right now. 
please let me expand those thoughts before we'll start a discussion.

few technologies - meant React, Surface & Flutter

**Review by tkullisaar — APPROVED** [review]

**Pull request merged**